src/app.js

```diff
@@ -6,6 +6,9 @@ var vorpal = require('vorpal')();
 //GameManager object
 var GameManager = new (require('./classes/GameManager.js'))();
 
```

Comment:
Go more detail of what the variable is doing. I know its self explanatory but just in case.

Reply:
Added more detail and resolved.